### PR TITLE
4-add-signinsignup-modules-2 

### LINKED ISSUE #4: Add: SignIn/SignUp Modules
Is your Feature Request Related to a Problem? Please Describe.
N/A

---------------

Describe how the solution should behave
The modules should be inside the components directory and called inside the navbar because they will be accessible from there.

----------------

Screenshots of the design/if any

----------------

Additional context
Use formik to handle validation.

### PR DESCRIPTION
**Pull request by goksu1**
#4 
Login signup pages were created with their responsive structure.
tailwind and formik are used.
login1

Signup1
responsivelogin
responsivesignup

.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
+    "plugins": [
+        "react",
+        "prettier"
+    ],
+    "rules": {
+        "quotes":["error","single",{"avoidEscape": true}],
+        "react/jsx-uses-react":["off"],
+        "react/react-in-jsx-scope":["off"],
+        "react/jsx-props-no-spreading":["off"],
+        "jsx-a11y/label-has-associated-control":["off"],
```
Comment (Iseluin) [inline .eslintrc.json:29]: Please delete this exception. We do not have this and we do not add exceptions.

**Review by maqalaqil — APPROVED** [review]
LGTM  @goksu1 
only a small comment please center the design for mobile. 
For future PRs please try to add visuals and keep your branch up to date

File: src/routes.js [hunk 2]
```diff
@@ -6,3 +6,5 @@ export const LOG_IN = '/login';
 export const LOG_OUT = '/logout';
 export const DONATION = '/donation';
 export const REQUEST = '/request';
+export const Login = '/Login';
+export const Signup = '/Signup';
```
Comment (Iseluin) [inline src/routes.js]: Please follow the naming convention. SIGN_IN and SIGN_UP.

**Review by Iseluin — CHANGES_REQUESTED** [review]

File: src/routes.js (hunk 2 above)
Comment (goksu1) [reply src/routes.js]: I fixed it

**Pull request merged**